content/en/docs/setup/install/multicluster/virtual-machine/index.md

```diff
@@ -0,0 +1,141 @@
+1.  Virtual machines must have IP connectivity to the ingress gateway in the connecting mesh, or alternatively, every pod in the mesh via L3 networking.
+
+## Installing the Istio control plane
+
+1. Generate intermediate CA certificates for each cluster's CA from your
+    organization's root CA. The shared root CA enables mutual TLS communication
+    across different clusters or virtual machines. For illustration purposes,
+    the following instructions use the certificates from the Istio samples
+    directory for the cluster.
+
+1. Run the following commands on each cluster in the mesh where a virtual
+    machine connection should be established. See
+    [Certificate Authority (CA) certificates](/docs/tasks/security/cert-management/plugin-ca-cert/)
+    for more details on configuring an external CA.
+
+    {{< text bash >}}
+    $ cd samples/certs
+    $ make vm
+    $ kubectl create namespace istio-system
+    $ kubectl create secret generic cacerts -n istio-system \
+        --from-file=vm/ca-cert.pem \
+        --from-file=vm/ca-key.pem \
+        --from-file=vm/root-cert.pem \
+        --from-file=vm/cert-chain.pem
+    $ cd ../..
+    {{< /text >}}
+
+1. Generate a `cluster.env` configuration file that informs the virtual machine
+   deployment which network CIDR to capture and redirect to the Kubernetes
+   cluster:
+
+    {{< text bash >}}
+    $ ISTIO_SERVICE_CIDR=$(echo '{"apiVersion":"v1","kind":"Service","metadata":{"name":"tst"},"spec":{"clusterIP":"1.1.1.1","ports":[{"port":443}]}}' | kubectl apply -f - 2>&1 | sed 's/.*valid IPs is //')
+    $ echo -n ISTIO_SERVICE_CIDR=$ISTIO_SERVICE_CIDR > $HOME/cluster.env
+    {{< /text >}}
+
+1. Configure a select set of ports for exposure from the virtual machine. If you do not
+   apply this optional step, all outbound traffic on all ports is sent to the
+   Kubernetes cluster. You may wish to send some traffic on specific ports to
+   other destinations. This example shows enabling ports `3306` and `8080`
+   for capture by the Istio virtual machine integration and transmission to Kubernetes.
+   All other ports are sent over the default gateway of the virtual machine.
+
+    {{< text bash >}}
+    $ echo "ISTIO_INBOUND_PORTS=3306,8080" >> $HOME/cluster.env
+    {{< /text >}}
+
+1. Install Istio with mesh expansion features:
+
+    {{< text bash >}}
+    $ cat <<EOF> vmintegration.yaml
+    apiVersion: install.istio.io/v1alpha1
+
+    kind: IstioOperator
+    spec:
+      values:
+        global:
+          meshExpansion:
+            enabled: true
+          controlPlaneSecurityEnabled: true
+    EOF
+    $ istioctl install -f vmintegration.yaml
+    {{< /text >}}
+
+### Installing the virtual machine
+
+Run the following commands on the virtual machine you want to add to the Istio mesh:
+
+1. Copy the previously created `cluster.env` and `*.pem` files to the virtual machine.
+
+1. Install the Debian package containing the Istio virtual machine integration runtime.
```

Comment:
Are other distributions supported (e.g. RedHat/CentOS)? I'm not too familiar with VM support. If so I'd link to them as well, but if not I'd put something at the top saying only Debian based distros are supported (Debian, Ubuntu, etc..).

Reply:
They are not (yet). Its a 1.7 roadmap item

Reply:
I am attempting to keep the content devoid of vendor-specific content. I am happy to mention "There is only DEB packaging available but no RPM packaging" if it really adds value. RPM packaging is in plan for 1.7 as @howardjohn has mentioned, although generally, we don't make forward-looking statements on the website.

Reply:
resolving for now. @jacob-delgado if you agree with my no-action on this suggestion, please click "Show resolved" and "Unresolve conversation.

Cheers
-steve